lib/filters.js

```diff
@@ -51,6 +58,17 @@ function httpHeaders (payload) {
         .map(function (k) { return k + '=' + cookies[k] })
         .join('; ')
     }
+
+    if (typeof headers['set-cookie'] !== 'undefined') {
+      try {
+        var setCookies = new SetCookie(headers['set-cookie'])
+        redact.forEach(setCookies)
+        headers['set-cookie'] = stringify(setCookies)
+      } catch (err) {
+        // Ignore error
+        headers['set-cookie'] = '[malformed set-cookie header]'
```

Comment:
Are we sure we want to overwrite it in this case? I guess it's to prevent leaking secrets, but it does make it hard to fix it if the user never sees how it looked 🤔

Reply:
@elastic/apm-agent-devs @roncohen: Should we redact the entire `Set-Cookie` header if we can't parse it or just report it as is?

Reply:
In the Java agent, we always redact the whole value. I’m not sure if the overhead of parsing the value is worth it.